eclair-core/src/main/scala/fr/acinq/eclair/channel/Monitoring.scala

```diff
@@ -21,6 +21,7 @@ import kamon.Kamon
 object Monitoring {
 
   object Metrics {
+    val ChannelCount = Kamon.gauge("channels.count")
```

Comment:
Nit: `ChannelsCount` or `ChannelCount`?

Reply:
I'll align with peer (and potentially update the metrics that don't follow the same pattern).